makeflow/src/mf_mesos_executor

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2016- The University of Notre Dame.
+# This software is distributed under the GNU General Public License.
+# See the file COPYING for details. 
+
+# This script defined a customized mesos executor. Compare to the default executor, 
+# it can send the url of sandbox directory of each executor to the scheduler. And
+# then the Mesos scheduler is able to retrieve the outputs. 
+
+import os
+import sys
+import json
+import urllib2
+import shutil
+import logging
+import threading
+import subprocess
+
+sys.path.insert(0, sys.argv[4])
+
+from mesos.native import MesosExecutorDriver
+from mesos.interface import Executor
+from mesos.interface import mesos_pb2
+
+logging.basicConfig(filename=('{0}.log'.format(sys.argv[2])), 
+                    level=logging.INFO,
+                    format='%(asctime)s %(name)-12s %(levelname)-8s %(message)s',
+                    datefmt='%m-%d %H:%M',)
+
+DEFAULT_PORT = 5051
+
+class MakeflowMesosExecutor(Executor):
+
+    def __init__(self, cmd, executor_id, framework_id, hostname):
+        self.hostname = hostname
+        self.port = DEFAULT_PORT
+        self.cmd = cmd
+        self.executor_id = executor_id
+        self.framework_id = framework_id
+
+    def disconnected(self, driver):
+        driver.sendFrameworkMessage("[EXUT_STATE] {0} {1} disconnected".format(self.executor_id, self.task_id))
```

Comment:
Is that a mesos thing? If not, that will look like a typo to people in the future.  EXECUTOR_STATE is better.